app/views/new_user.erb

```diff
@@ -0,0 +1,7 @@
+<form id="new_user" action="/users" method="post">
+  <input type="text" placeholder="email" name="email"><br>
+  <input type="text" placeholder="username" name="username"><br>
+  <input type="password" placeholder="password" name="password"><br>
+  <input type="submit" value="Register">
+</form>
+<%= session[:errors] %>
```

Comment:
instead of the session use an instance variable like `@user`. This works if you render right away instead of redirect